echemdb/database.py

```diff
@@ -60,6 +60,9 @@ class Database:
         0
 
     """
+    from echemdb.entry import Entry
+
+    Entry = Entry
 
```

Comment:
It looks like this is the point where you stopped. Consider this comment as a bookmark.

Reply:
True. I did not get much further this time either.